lib/stormpath-sdk/id_site/error.rb

```diff
@@ -0,0 +1,42 @@
+module Stormpath
+  module IdSite
+    class Error < Stormpath::Error
```

Comment:
Yes, these where my initial thoughts as well. 
But on handle_id_site_callback method we can’t capture errors from the server we only get a response_uri which containes the jwt token. After the token is decoded we add some checks for errors based on the decoded token

I was checking node SDK and found the same implementation here
https://github.com/stormpath/stormpath-sdk-node/blob/master/lib%2Fresource%2FApplication.js#L103